.github/workflows/deployment-staging.yml

```diff
@@ -0,0 +1,30 @@
+# The Purpose of this workflow file is to enable us to automatically 
+# send a build request for the staging repository based 
+# on the latest commit from upstream (CATcher-org/CATcher)
```

Comment:
Minor nitpick: There isn't a need to mention that the build is based "on the latest commit from upstream" here right?
This deployment workflow is conceptually independent from the synchronisation workflow.

Reply:
Yeps! That is right to say that!

Reply:
Alright, then let's change the comment to "automatically deploy the app based on the latest commit on the staging repo"

Reply:
Alright!